Tests/Marketplace/marketplace_services.py

```diff
@@ -330,6 +344,24 @@ def _get_all_pack_images(pack_integration_images, display_dependencies_images, d
 
         return pack_integration_images
 
+    def is_feed_pack(self, yaml_content, yaml_type):
+        """
+        Checks if an integration is a feed integration. If so, updates Pack._is_feed
+        Args:
+            yaml_content: The yaml content extracted by yaml.safe_load().
+            yaml_type: The type of object to check. Should be 'Playbook' or 'Integration'.
+
+        Returns:
+            Doesn't return
+        """
+        if yaml_type == 'Integration':
+            for configuration in yaml_content.get('configuration'):
```

Comment:
Why are you searching in `configuration` section in the yml and not under `script` -> `feed` == true?

Reply:
@IkaDemisto no actual reason. I didn't know this field. I just knew that every feed integration has a `Fetch Indicators` command. Will change to `script`->`feed` check.